test/e2e/app/main.go

```diff
@@ -103,6 +104,7 @@ func startNode(cfg *Config) error {
 		proxy.NewLocalClientCreator(app),
 		node.DefaultGenesisDocProviderFunc(tmcfg),
 		node.DefaultDBProvider,
+		ipfs.Mock(), // TODO(ismail): Should we use real IPFS for E2E?
```

Comment:
BTW, we use TODO(name) with the name of the author of the TODO. This is in order to able to ask the creator of the todo what they intended to be done. So this should be:
```suggestion
		ipfs.Mock(), // TODO(Wondertan): Should we use real IPFS for E2E?
```

To answer your question: we currently do not use IPFS in the e2e tests. So for now it does not really matter. A real object would just take more time when initializing it and we won't really make use of to. So a mock object is fine for now. 

The moment we actually want to test ipfs-related methods in the e2e tests (e.g. `ValidateAvailability` via the light client), it needs to be a real ipfs object. We first need to make sure that the different nodes running in different containers are able to actually find each other.